vector/src/main/java/im/vector/app/features/call/audio/API21AudioDeviceDetector.kt

```diff
@@ -25,9 +25,12 @@ import android.media.AudioManager
 import androidx.core.content.getSystemService
 import im.vector.app.core.services.BluetoothHeadsetReceiver
 import im.vector.app.core.services.WiredHeadsetStateReceiver
+import org.matrix.android.sdk.api.logger.LoggerTag
 import timber.log.Timber
 import java.util.HashSet
 
+private val loggerTag = LoggerTag("API21AudioDeviceDetector", LoggerTag.VOIP)
```

Comment:
Why not using `API21AudioDeviceDetector::class.simpleName` instead of hard coded string? For proguard?

Reply:
Yup